containers/flatpak/cockpit-client.yml.in

```diff
@@ -0,0 +1,38 @@
+app-id: @FLATPAK_ID@
+runtime: org.gnome.Platform
+runtime-version: '41'
```

Comment:
How long are these supported? I.e. how often do we need to set an alarm clock to bump this? Just the 6 months, or do they support multiple versions at a time?

Reply:
They support multiple versions for a while.